bigtable/bigtable.go

```diff
@@ -453,9 +461,13 @@ func prefixSuccessor(prefix string) string {
 	return string(ans)
 }
 
+// RequestStatsCallback describes a callback that receives a RequestStats for evaluation.
+type RequestStatsCallback func(*btpb.RequestStats)
+
 // A ReadOption is an optional argument to ReadRows.
 type ReadOption interface {
 	set(req *btpb.ReadRowsRequest)
+	getRequestStatsCallback() RequestStatsCallback
```

Comment:
If I read this correctly, every distinct type that implements the ReadOption interface must now have the `getRequestStatsCallback()` function. As this is a private method, it won't cause a breaking change in the API surface, but it will require refactoring all existing ReadOption types.